plugins/urls/__init__.py

```diff
@@ -335,343 +355,151 @@ def _respond_shorten(self, result, source, handler):
 
         if result is not None:
             return source.respond(result)
-        return source.respond(__("Unable to shorten using handler %s. Poke the"
-                                 "bot owner!")
-                              % handler)
+
+        source.respond("Unable to shorten using handler %s. Poke the bot "
+                       "owner!" % handler)
 
     def _respond_shorten_fail(self, failure, source, handler):
         """
         Respond to a shorten command, after a failed Deferred
         """
 
-        return source.respond(__("Error shortening url with handler %s: %s")
-                              % (handler, failure))
+        source.respond("Error shortening url with handler %s: %s" % (
+            handler, failure
+        ))
 
     def shorten_command(self, protocol, caller, source, command, raw_args,
-                        parsed_args):
+                        args):
         """
         Command handler for the shorten command
         """
 
-        args = parsed_args  # Quick fix for new command handler signature
         if not isinstance(source, Channel):
             if len(args) == 0:
-                caller.respond(__("Usage: {CHARS}shorten [url]"))
+                caller.respond("Usage: {CHARS}shorten [url]")
                 return
             else:
-                handler = "tinyurl"
-                url = args[0]
+                handler = self.default_shortener
+                _url = args[0]
                 try:
-                    d = self.shorten_url(url, handler)
+                    d = self.shorten(_url, handler)
                     d.addCallbacks(self._respond_shorten,
                                    self._respond_shorten_fail,
                                    callbackArgs=(source, handler),
                                    errbackArgs=(source, handler))
                 except Exception as e:
-                    self.logger.exception(_("Error fetching short URL."))
-                    caller.respond(__("Error: %s") % e)
-                    return
+                    self.logger.exception("Error fetching short URL.")
+                    caller.respond("Error: %s" % e)
```

Comment:
Yeah, but it's nice for them to have the exception to give us for debugging, instead of having to restart in debug mode.

I mean I guess we have metrics, but not everyone enables that.